css/agency.css

```diff
@@ -792,26 +803,3 @@ img::-moz-selection {
 body {
   webkit-tap-highlight-color: #fed136;
 }
-
-/* Scroll to top of page */
```

Comment:
Why did you remove the scroll-top-top function?

Reply:
@charmaine Actually previously I was working on some other branch (since I'm a first-time contributor) and later I got to know I was supposed to fork the repo and make changes in that. Apologies. Will add it again and update the PR.